pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import functools
+import subprocess
+from subprocess import STDOUT
+
+import pwndbg.commands
+import pwndbg.which
+
+
+
```

Comment:
Afaik pep8 cries if there are more then 2 newlines; guess isort doesn't check it ;f